### ISSUE TITLE
`__DEV__ is not defined` error in React Native Web

### ISSUE THREAD

**Issue by crutchcorn** [codemotionapps/react-native-dynamic#7]
While trying to use the project with React Native Web (namely, Storybooks), I get the following errors:

```
ReferenceError: __DEV__ is not defined
    at parseStylesFor (http://localhost:6006/vendors~main.4e9459873265b87a02ad.bundle.js:9659:1056)
    at new DynamicStyleSheet (http://localhost:6006/vendors~main.4e9459873265b87a02ad.bundle.js:9659:1359)
    at Object../src/components/shark-checkbox/shark-checkbox.tsx (http://localhost:6006/main.4e9459873265b87a02ad.bundle.js:614:2792)
    at __webpack_require__ (http://localhost:6006/runtime~main.4e9459873265b87a02ad.bundle.js:849:30)
    at fn (http://localhost:6006/runtime~main.4e9459873265b87a02ad.bundle.js:151:20)
    at Object.<anonymous> (http://localhost:6006/main.4e9459873265b87a02ad.bundle.js:602:914)
    at Object../src/components/shark-checkbox/shark-checkbox.stories.tsx (http://localhost:6006/main.4e9459873265b87a02ad.bundle.js:603:30)
    at __webpack_require__ (http://localhost:6006/runtime~main.4e9459873265b87a02ad.bundle.js:849:30)
    at fn (http://localhost:6006/runtime~main.4e9459873265b87a02ad.bundle.js:151:20)
    at webpackContext (http://localhost:6006/main.4e9459873265b87a02ad.bundle.js:91:9)
```

This seems to appear from this code: https://github.com/codemotionapps/react-native-dynamic/blob/master/library/src/dynamic-style-sheet.ts#L41

**Comment by crutchcorn** [comment]
It's worth mentioning that I was able to work around this issue by modifying the webpack config in `.storybook/webpack.config.js`

```
  config.plugins.push(
      new webpack.DefinePlugin({
        '__DEV__': JSON.stringify(true)
      })
  )
```

**Comment by dimitarnestorov** [comment]
Released 1.0.0. Should be fixed there.
